regression/esbmc/simplify-float-nan/test.desc

```diff
@@ -0,0 +1,4 @@
+CORE
+main.cpp
+--overflow-check
+^VERIFICATION SUCCESSFUL$
```

Comment:
BTW, shall we classify this test case as `VERIFICATION FAILED`?

This is the Valgrind output:

````
==24150== Memcheck, a memory error detector
==24150== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==24150== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==24150== Command: ./int
==24150==
==24150== Invalid read of size 1
==24150==    at 0x49A06C0: __strtof_nan (strtod_nan_main.c:37)
==24150==    by 0x1091C0: __static_initialization_and_destruction_0(int, int) (in /home/lucas/examples/int)
==24150==    by 0x1091F8: _GLOBAL__sub_I_a (in /home/lucas/examples/int)
==24150==    by 0x4979EBA: call_init (libc-start.c:145)
==24150==    by 0x4979EBA: __libc_start_main@@GLIBC_2.34 (libc-start.c:379)
==24150==    by 0x1090A4: (below main) (in /home/lucas/examples/int)
==24150==  Address 0x0 is not stack'd, malloc'd or (recently) free'd
==24150==
==24150==
==24150== Process terminating with default action of signal 11 (SIGSEGV)
==24150==  Access not within mapped region at address 0x0
==24150==    at 0x49A06C0: __strtof_nan (strtod_nan_main.c:37)
==24150==    by 0x1091C0: __static_initialization_and_destruction_0(int, int) (in /home/lucas/examples/int)
==24150==    by 0x1091F8: _GLOBAL__sub_I_a (in /home/lucas/examples/int)
==24150==    by 0x4979EBA: call_init (libc-start.c:145)
==24150==    by 0x4979EBA: __libc_start_main@@GLIBC_2.34 (libc-start.c:379)
==24150==    by 0x1090A4: (below main) (in /home/lucas/examples/int)
==24150==  If you believe this happened as a result of a stack
==24150==  overflow in your program's main thread (unlikely but
==24150==  possible), you can try to increase the size of the
==24150==  main thread stack using the --main-stacksize= flag.
==24150==  The main thread stack size used in this run was 8388608.
==24150==
==24150== HEAP SUMMARY:
==24150==     in use at exit: 0 bytes in 0 blocks
==24150==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==24150==
==24150== All heap blocks were freed -- no leaks are possible
==24150==
==24150== For lists of detected and suppressed errors, rerun with: -s
==24150== ERROR SUMMARY: 1 errors from 1 contexts (suppressed: 0 from 0)
Segmentation fault
````

Reply:
Valgrind seems to have crashed here. This example doesn't seem to have an overflow, but it can lead to undefined behavior?